### ISSUE TITLE
Weak Puppo training on current ml-agents version

### ISSUE THREAD

**Issue by Shinao** [Unity-Technologies/ml-agents#1597]
I'm trying to make the Puppo working on the new ml-agents version but I don't get very far with the training : 
`INFO:mlagents.trainers: DogBrain-0: Puppo: Step: 50000. Mean Reward: 3.024. Std of Reward: 1.362. Training.`

The docs says to install the 0.5 version but if I have to install a version for every ml-agent environment I want to try it's gonna be difficult, much better to know how to upgrade it I think.

Anyone from the Puppo original team can tell me what needs to be changed to make it work ?

Thanks

**Comment by vincentpierre** [comment]
From my memories, a reward around 3 should already give you a nice looking Puppo. There are some instruction on how to migrate a project from v0.5 to v0.6 [here](https://github.com/Unity-Technologies/ml-agents/blob/master/docs/Migrating.md#migrating-from-ml-agents-toolkit-v05-to-v06)

**Comment by Shinao** [comment]
I've found the problem, I thought the hyperparameters were used from the name of the argument --run-id but it's just taking the brain asset name. When upgrading to v0.6 I changed to "Puppo" instead of "DogBrain" and it took the Default path.
Now Puppo goes from a reward of 3 (sluggish movement) to 3.8 and it's the same as the v0.5.
I guess I should spend more time learning about this parameters from now on.

Thanks
